packages/@coorpacademy-components/src/molecule/video-iframe/index.js

```diff
@@ -47,18 +55,15 @@ class VideoIframe extends React.Component {
   render() {
     const {type, id, url, autoplay = false, width = '100%', height = '400px'} = this.props;
     const src = getUrl({url, type, id, autoplay});
-
+    const properties = {
+      allow: includes(['faurecia-vr'], type) ? 'xr-spatial-tracking;fullscreen' : null,
```

Comment:
c'est plutôt l'inverse, comme tu as fait plus bas?
`includes(value, collection)`
![image](https://github.com/CoorpAcademy/components/assets/23300140/a85eef00-7d73-4f62-88eb-95bfee451351)
 Quand je teste il supporte pas un array en 1er param
```suggestion
      allow: includes(type, ['faurecia-vr']) ? 'xr-spatial-tracking;fullscreen' : null,
```